expyfun/stimuli/_tracker.py

```diff
@@ -93,6 +94,10 @@ class TrackerUD(object):
         The minimum value that the tracker level (``x``) is allowed to take.
     x_max : float
         The maximum value that the tracker level (``x``) is allowed to take.
+    repeat_limit : str
+        How to treat trials that hit either x_min or x_max. ``reversals`` will
+        count these trials as reversals while staying at the same level.
+        ``ignore`` does not automatically count these trials as reversals.
```

Comment:
I think it's less about "How to treat trials that *hit* either x_min or x_max" and more about "How to treat trials that, *based on the responses, should exceed* either x_min or x_max". In other words, touching is no problem / not interesting. What needs to be decided is how to deal with what happens when a user response indicates that they should go in a direction that they cannot. Agreed?

Is this how you have it implemented in the code, namely that you only need to do something special (stay flat, and optionall count as a reversal depending on this ``kwarg``) if the tracker would otherwise try to exceed a limit?